faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -666,7 +666,7 @@ class ClientSpec
 
   }
 
-  it should "test miscellaneous functions" in {
+  it should "test miscellaneous functions" ignore {
```

Comment:
I am slightly uncomfortable because this test does a lot more than asserting `Contains`.
For how long is it supposed to be like that?

Reply:
Yeah, there's the problem where we're testing too many things under a single test case...

As soon as the stable image with the reintroduced `Contains` function is available, those tests are going to be reestablished as well. This is supposed to happen sometime next week. 

In any case, we are definitely going to reestablish the tests before doing a Driver release.